MozillaFoundation/foundation.mozilla.org
# Split integration tests into two files

Changed directory structure for integrations a bit.

From 
```
tests
├── integration.spec.js
├── ...
├── ...
├── ...

```
to

```
tests
├── integration
│   ├── foundation-homepage.spec.js
│   └── pni.spec.js
├── ...
```

Related PRs/issues: #10618 

### To test

1. on `main` branch, run `docker-compose up`
2. on a different terminal tab, run `npm run playwright:integration`
3. observe results
4. switch to this PR, run `docker-compose up`
5. on a different terminal tab, run `npm run playwright:integration`
6. observe results to make sure it matches what you see on Step 3
7. If results are the same, PR is good to go!
